libs/clients/national-registry/v3/src/lib/types/address.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { EinstaklingurDTOHeimili } from '../../../gen/fetch'
```

Comment:
since these are all dto's, then I would recommend renaming the types folder to dto